<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditDoctor</class>
 <widget class="QDialog" name="AddEditDoctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Учетная запись врача</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/Forms/Icons/Forms/Icons/identity.svg</normaloff>:/Forms/Icons/Forms/Icons/identity.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Учетная запись врача</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Имя пользователя</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="UsernameField"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="PasswordField"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="LastnameField"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="FirstnameField"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Отчество</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="PatronymicField"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Специальность</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="SpecialityField"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Дата найма</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDateEdit" name="EmploymentDateField"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="AcceptButton">
     <property name="text">
      <string>Принять</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
